firebase_admin/ml.py

```diff
@@ -785,9 +786,15 @@ def __init__(self, app):
         self._project_url = _MLService.PROJECT_URL.format(self._project_id)
         self._client = _http_client.JsonHttpClient(
             credential=app.credential.get_credential(),
+            headers={
```

Comment:
Nit: Avoid duplication. Initialize the headers into a variable, and pass it to the constructors.

Reply:
done.